src/detailed/introduction.md

```diff
@@ -0,0 +1,45 @@
+# Eludris Detailed
+Dedicated and detailed rundown of how we *plan* Eludris to eventually function.
+This is mostly sketched out from and is and extension of the old `plan.md` in our `eludris/eludris` repository.
+
+## Overview
+
+The goal is to provide a uniquely fresh but not totally new experience focused around any topic or interest whilst having a policy of being secure, free (as in monetary value), free (as in freedom), privacy respecting, open source, and decentralized without putting users who may not have knowledge or anything to do with the aforementioned topics at an inconvenience.
+
+In addition, as with anything Eludris-related, modifying the source code or making your own clients, libraries, and tooling around Eludris is more than welcome.
+
+## Takeaways
+
+### You can make Communities.
+
+Communities can be either message-based, post-based, or both.
+
+Message-based communities work like how a Discord server does, having many different channels secluded to their own types and members can -as you can guess- create messages within channels.
```

Comment:
I would split this into two sentences, too:


Message-based communities work like how a Discord server does, having many different channels secluded to their own types. Members can -as you can guess- create messages within channels.